.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,17 +8,4 @@ about: Create a report to help us improve
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Steps to reproduce the behavior:
```

Comment:
I'm personally a big fan of the PR template of HyperSpy ([here](https://github.com/hyperspy/hyperspy/blob/RELEASE_next_minor/.github/PULL_REQUEST_TEMPLATE.md)), which we have adopted in orix/kikuchipy ([here](https://github.com/pyxem/orix/blob/develop/.github/pull_request_template.md)). Did you consider these when making these changes?

Reply:
I felt that it's rather a lot for a new user to come up against. Matter of taste I think, but either this or thiers is better than what we currently have.

Reply:
I agree that it might be intimidating for a first time contributor. I'm OK with leaving it as it is.